targets/UnrealMarketplacePlugin/examplesource/ExampleProject/Plugins/PlayFab/Source/PlayFab/Public/Tests/PlayFabBlueprintTests.h.ejs

```diff
@@ -21,9 +21,13 @@ class PLAYFAB_API UPlayFabBlueprintTests : public UPlayFabTestCase
 
 public:
     int testMessageInt;
+    UPROPERTY()
```

Comment:
nit: newline above UPROPERTY

Reply:
Every other adjacent file keeps the class properties together without breaks between.
Just being consistent.